### PR TITLE
Add rights

### PR DESCRIPTION
**Pull request by vakyl9r**
Add rights to admin/user (maybe moderator)

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -9,6 +9,7 @@ def show
   end
 
   def edit
+    authorize @comment
```
Comment (Magons) [inline app/controllers/comments_controller.rb]: move to separate method

File: app/policies/comment_policy.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class CommentPolicy < ApplicationPolicy
+  attr_reader :current_user, :comment
+
+  def initialize(current_user, comment)
+    @current_user = current_user
+    @comment = comment
+  end
+
+  def update?
+      @current_user.try(:admin?) or @current_user.try(:moderator?) or (@comment.user_id == @current_user and @current_user.try(:user?))
```
Comment (Magons) [inline app/policies/comment_policy.rb]: use `||`.  instead `try` use `&` `@current_user&.admin?`

File: app/policies/post_policy.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+  def create?
+    @current_user.try(:admin?)
+  end
+
+  def update?
+    @current_user.try(:admin?) or @current_user.try(:moderator?)
+  end
+
+  def destroy?
```
Comment (Magons) [inline app/policies/post_policy.rb]: only admin can destroy post? can author  destroy?

**Comment by Magons** [comment]
@vakyl9r rebasse pls

**Comment by vakyl9r** [comment]
@Magons could you, please, take a look)

File: db/schema.rb [hunk 4]
```diff
@@ -15,14 +15,6 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "attachments", force: :cascade do |t|
```
Comment (Magons) [inline db/schema.rb:18]: why remove? where is the migration?

File: app/views/posts/index.html.haml [hunk 5]
```diff
@@ -25,12 +25,14 @@
...
-          %td.no-border
-            =link_to edit_post_path(post), class: 'btn btn-default icon' do
-              %span.glyphicon.glyphicon-pencil(aria-hidden= 'true')
-          %td.no-border
-            =link_to post, method: :delete, data: { confirm: 'Вы уверены?' }, class: 'btn btn-default icon' do
-              %span.glyphicon.glyphicon-trash(aria-hidden= 'true')
+          - if policy(post).edit?
+            %td.no-border
+              =link_to edit_post_path(post), class: 'btn btn-default icon' do
+                %span.glyphicon.glyphicon-pencil(aria-hidden= 'true')
```
Comment (Magons) [inline app/views/posts/index.html.haml:31]: does it work? `(aria-hidden= 'true')`

Comment (vakyl9r) [reply app/views/posts/index.html.haml:31]: seems to work fine. actually, i don't have a clue, i didn't write this)

**Pull request merged**